keyboards/boardwalk/keymaps/niclake/keymap.c

```diff
@@ -37,7 +37,7 @@ enum custom_keycodes {
 #define FN MO(_FN)
 #define NUM TT(_NUM)
 #define ADJ MO(_ADJ)
-#define MACWIN AG_TOGG
+#define MACWIN MAGIC_TOGGLE_ALT_GUI
```

Comment:
This is the same keycode, in theory.

Reply:
I know, this was for my own well-being of being able to remember why I have `BOOTMAGIC_ENABLE` turned on (as it's easier for me to search for across my project). I only changed my own define here.